scargo/commands/test.py

```diff
@@ -152,3 +156,64 @@ def run_it(config: Config, verbose: bool) -> None:
 
     subprocess.check_call(cmd)
     subprocess.check_call("cat it-coverage.txt", shell=True)
+
+
+def conan_add_remote(project_path: Path, config: Config) -> None:
```

Comment:
We don't want those functions in test.py, I wanted only **function calls** in the test command and function definitions in build.py. Even in build.py is not an ideal place, but it should be better than test.py. 
If you want, you can create some kind of conan_utils.py file in the same place where clang_utils.py is and other util files.